src/um/fltuser.rs

```diff
@@ -0,0 +1,376 @@
+    FLT_FSTYPE_RAW,             //Microsoft's RAW file system       (\FileSystem\RAW)
+    FLT_FSTYPE_NTFS,            //Microsoft's NTFS file system      (\FileSystem\Ntfs)
+    FLT_FSTYPE_FAT,             //Microsoft's FAT file system       (\FileSystem\Fastfat)
+    FLT_FSTYPE_CDFS,            //Microsoft's CDFS file system      (\FileSystem\Cdfs)
+    FLT_FSTYPE_UDFS,            //Microsoft's UDFS file system      (\FileSystem\Udfs)
+    FLT_FSTYPE_LANMAN,          //Microsoft's LanMan Redirector     (\FileSystem\MRxSmb)
+    FLT_FSTYPE_WEBDAV,          //Microsoft's WebDav redirector     (\FileSystem\MRxDav)
+    FLT_FSTYPE_RDPDR,           //Microsoft's Terminal Server redirector    (\Driver\rdpdr)
+    FLT_FSTYPE_NFS,             //Microsoft's NFS file system       (\FileSystem\NfsRdr)
+    FLT_FSTYPE_MS_NETWARE,      //Microsoft's NetWare redirector    (\FileSystem\nwrdr)
+    FLT_FSTYPE_NETWARE,         //Novell's NetWare redirector
+    FLT_FSTYPE_BSUDF,           //The BsUDF CD-ROM driver           (\FileSystem\BsUDF)
+    FLT_FSTYPE_MUP,             //Microsoft's Mup redirector        (\FileSystem\Mup)
+    FLT_FSTYPE_RSFX,            //Microsoft's WinFS redirector      (\FileSystem\RsFxDrv)
+    FLT_FSTYPE_ROXIO_UDF1,      //Roxio's UDF writeable file system (\FileSystem\cdudf_xp)
+    FLT_FSTYPE_ROXIO_UDF2,      //Roxio's UDF readable file system  (\FileSystem\UdfReadr_xp)
+    FLT_FSTYPE_ROXIO_UDF3,      //Roxio's DVD file system           (\FileSystem\DVDVRRdr_xp)
+    FLT_FSTYPE_TACIT,           //Tacit FileSystem                  (\Device\TCFSPSE)
+    FLT_FSTYPE_FS_REC,          //Microsoft's File system recognizer (\FileSystem\Fs_rec)
+    FLT_FSTYPE_INCD,            //Nero's InCD file system           (\FileSystem\InCDfs)
+    FLT_FSTYPE_INCD_FAT,        //Nero's InCD FAT file system       (\FileSystem\InCDFat)
+    FLT_FSTYPE_EXFAT,           //Microsoft's EXFat FILE SYSTEM     (\FileSystem\exfat)
+    FLT_FSTYPE_PSFS,            //PolyServ's file system            (\FileSystem\psfs)
+    FLT_FSTYPE_GPFS,            //IBM General Parallel File System  (\FileSystem\gpfs)
+    FLT_FSTYPE_NPFS,            //Microsoft's Named Pipe file system(\FileSystem\npfs)
+    FLT_FSTYPE_MSFS,            //Microsoft's Mailslot file system  (\FileSystem\msfs)
+    FLT_FSTYPE_CSVFS,           //Microsoft's Cluster Shared Volume file system  (\FileSystem\csvfs)
+    FLT_FSTYPE_REFS,            //Microsoft's ReFS file system      (\FileSystem\Refs or \FileSystem\Refsv1)
+    FLT_FSTYPE_OPENAFS,         //OpenAFS file system               (\Device\AFSRedirector)
+    FLT_FSTYPE_CIMFS,            //Composite Image file system       (\FileSystem\cimfs)
+}}
+ENUM!{enum FILTER_INFORMATION_CLASS {
+    FilterFullInformation = 0,
+    FilterAggregateBasicInformation,
+    FilterAggregateStandardInformation,
+}}
+ENUM!{enum FILTER_VOLUME_INFORMATION_CLASS {
+    FilterVolumeBasicInformation = 0,
```

Comment:
```suggestion
    FilterVolumeBasicInformation,
```

Windows header file doesn't specify = 0.